ntc_templates/templates/index

```diff
@@ -506,3 +506,5 @@ watchguard_firebox_show_arp.textfsm, .*, watchguard_firebox, sh[[ow]] arp
 
 yamaha_show_environment.textfsm, .*, yamaha, sh[[ow]] en[[vironment]]
 yamaha_show_ip_route.textfsm, .*, yamaha, sh[[ow]] ip ro[[ute]]
+
+zyxel_os_cfg_lan_get_--Name_name.textfsm, .*, zyxel_os, cfg lan get --Name .+
```

Comment:
Similar to previous, does "cfg lan get" work?

Reply:
Unfortunately not. Much more information will be provided when fetching a specific lan info

Reply:
Let's at least make it `\S+`, presuming there can only be a name that does not include a space

Reply:
I managed to create one with space... No special characters tho. Maybe just a `[\s\w-]+` ?